apps/og-image/app/components/logo3d.tsx

```diff
@@ -13,9 +13,9 @@ export function Logo3d() {
           fill="#3178C6"
         />
         <path
-          fillRule="evenodd"
-          clipRule="evenodd"
-          d="M261.441 348.723H333.158V299.842H133.263V348.723H204.629V566.368H261.441V348.723Z"
+          fill-rule="evenodd"
+          clip-rule="evenodd"
```

Comment:
Why?

Reply:
@PickleNik it has to be camelcase in react! `fillRule` `clipRule`

Reply:
automatically copypasted from figma, fixed